src/lib.spec.js

```diff
@@ -103,8 +103,8 @@ describe("dirParseSync", () => {
 let exp = {
     "/": ["test-inside-root.txt"],
     folderA: ["test-inside-a.txt"],
-    "folderA/folderB": ["test-inside-b.txt"],
```

Comment:
hmm. I don't really like path.join being passed one variable - its job really is to handle multiple arguments.

Can you provide more details of the windows experience and what is not working?